src/config.js

```diff
@@ -30,6 +32,11 @@ export const flags = {
     description: 'Improved sidebar performance',
     default: false,
     icon: '😴'
+  },
+  code: {
+    description: 'Show the code panel',
+    default: false,
+    icon: '{}'
```

Comment:
Could also use #️⃣ if you wanted to stick with emoji but I think the brackets work well

Reply:
Yeah I dug around the emojis but none are quite as nice as the curlys :)